config/routes.rb

```diff
@@ -653,6 +653,9 @@
     namespace :reports do
       resources :hesa_exports, only: :show, path: 'hesa-exports', param: :year, constraints: ValidRecruitmentCycleYear
       resources :hesa_exports, only: :index, path: 'hesa-exports'
+      resources :providers, only: [], path: '' do
```

Comment:
What do these empty `:only` and `:path` params do?

Reply:
Its so we can shorten what the url to remove the /providers/ rails defaults to Without path: '' => 
/provider/reports/providers/:provider_id/active_applications_status(.:format)
with path: '' => 
/provider/reports/:provider_id/active_applications_status(.:format)

Reply:
Would an explicit route be better in this case ie.
```
get '/provider/reports/:provider_id/active-applications-status' => `reports#show`
```
as we've departed from the conventions of resourceful routing with the empty params anyway (so we don't really want to use the `resources` block for this.

Reply:
We plan to use the resources structure for the remaining provider relevant reports, so having an existing resource makes sense

Reply:
Will those changes be made in this PR? If not I think it's best to use the resource routing pattern as and when the code needs it.